inbm/dispatcher-agent/dispatcher/source/source_command.py

```diff
@@ -120,7 +120,7 @@ def _handle_app_source_command(
             logger.info(f"Optional GPG key parameters not present in manifest")
         filename = parsed_head.get_children("applicationSource/remove/repo")["filename"]
         application_source_manager.remove(
-            ApplicationRemoveSourceParameters(file_name=filename, gpg_key_name=keyname)
+            ApplicationRemoveSourceParameters(source_list_file_name=filename, gpg_key_name=keyname)
         )
         return Result(status=200, message="SUCCESS")
```

Comment:
Use a constant for the HTTP status code `200` to improve code readability and maintainability. [medium]